lua/utils/init.lua

```diff
@@ -88,11 +88,16 @@ function utils.toggle_autoformat()
 end
 
 function utils.reload_lv_config()
+  -- Hot reload of statusline
   require("core.lualine").config()
-  vim.cmd "source ~/.local/share/lunarvim/lvim/lua/settings.lua"
+
+  -- Remove package cache
+  package.loaded["settings"] = nil
+  package.loaded["plugins"] = nil
```

Comment:
Not necessary, the file won't change

Reply:
Actually no, plugins here is necessary in case the user changes the `active` options of a builtin.

Reply:
Indeed, we should probably wrap data that can be reloaded into functions, another thing to add to my notes :)